feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -236,12 +236,16 @@ func (tc *testCase) verifyInstall(ctx context.Context, t *testing.T) {
 		default:
 			t.Fatalf("OS.Verify request failed: %v", err)
 		}
-
+		// when noreboot is set to false, the device returns "in-progress" before initiating the reboot
+		if got, want := r.GetActivationFailMessage(), "in-progress"; got == want {
```

Comment:
Here we don't know if are expecting "in-progress". So, to me it's just a normal condition that if we see it we sleep 1 min and then continue. Therefore, I think it make sense to just say `if r.GetActivationFailMessage() == "in-progress"`.